sidebars.json

```diff
@@ -35,7 +35,8 @@
             "concepts-broker-load-balancing-features",
             "concepts-broker-load-balancing-benefits",
             "concepts-broker-load-balancing-concepts",
-            "concepts-broker-load-balancing-types"
+            "concepts-broker-load-balancing-types",
+            "concepts-broker-load-balancing-quick-start"
```

Comment:
For reviewers: I know here is not the perfect place, but there is no better place to locate this quick start, so I put all types of docs under the same subject "Broker Load Balancing". I'll adjust it once there is a better solution.